biojava-structure/src/main/java/org/biojava/nbio/structure/align/ce/OptimalCECPMain.java

```diff
@@ -507,7 +504,7 @@ public AFPChain alignOptimal(Atom[] ca1, Atom[] ca2, Object param, AFPChain[] al
 			AFPChain currentAlignment = alignPermuted(ca1,ca2p,param,cp);
 
 			// increment progress bar
-			if(debug) System.out.print(".");
+			LOGGER.debug(".");
```

Comment:
Here the output won't be as before (a row of dots in same line), instead it will print a dot per log line. So it won't be a 'progress bar' anymore.

Suggestions anyone on how to better do this one with logging?